doc/Caliptra.md

```diff
@@ -1060,59 +1060,59 @@ As NIST publishes new standards with PQC resilience, algorithms applicable to Ca
 
 Firmware signing key rotation shall follow the requirements described in [Reference 3](#ref-3).
 
-# Hardware Section
+# Hardware section
 
-Caliptra Core Block Diagram
+The following figure describes the Caliptra Core.
 
-*Figure 9: Core Block Diagram*
+*Figure 9: Caliptra Core block diagram*
 
 ![](./images/Caliptra_HW_diagram.png)
 
 * SRAM requirements:
   * 128 KiB of ICCM0
   * 128 KiB for Mailbox as a staged SRAM (for FW staging of impactless updates to do authentication checks on the FW before moving to ICCM)
   * 128 KiB for DCCM and 32 KiB for ROM
-* Crypto requirements
+* Crypto requirements:
   * SHA256 SHA384, SHA512
   * ECC Secp384r1 w/ HMAC-DRBG - Key Generation, Sign and Verification
   * HMAC SHA384
   * AES256-ECB, CBC, GCM
-* Chips Alliance is used for RISC-V
-* APB is the choice of SOC facing interface
-* JTAG is exported at the IP interface
-* TRNG is the digital logic and algorithms required for random number generation. It requires a physical entropy source input. Refer to the Caliptra IP specification and integration specification for more information.
-  * For SOCs that want to use their own legacy TRNG, Caliptra provides a HW API to push the random number on request/response handshake. Refer to the ase see Caliptra IP specification and integration specification for more information.
+* Chips Alliance is used for RISC-V.
+* APB is the choice for the SoC-facing interface.
+* JTAG is exported at the IP interface.
+* TRNG is the digital logic and algorithms that are required for random number generation. It requires a physical entropy source input. See the Caliptra IP specification and integration specification for more information.
+  * For SoCs that want to use their own legacy TRNG, Caliptra provides a HW API to push the random number on the request/response handshake. See the Caliptra IP specification and integration specification for more information.
 
-Caliptra IP HW Boot Flow
+The following figure describes the Caliptra IP HW boot flow.
 
-*Figure 10: Hardware Boot Flow*
+*Figure 10: Hardware boot flow*
 
 ![](./images/Caliptra_boot_flow2.png)
 
-1. As a part of SOC boot flow, SOC may have a bunch of infrastructure and other entities that boot. That part of the flow is outside the scope of this document. If SOC chooses to bypass Caliptra, then it should have a capability to bypass the Caliptra entirely through its proprietary flow. This may be needed for A0 power on and other early validation.
+1. As part of the SoC boot flow, SoC may have other infrastructure and entities that boot. That part of the flow is outside the scope of this document. If SoC chooses to bypass Caliptra, then it should have a capability to bypass Caliptra entirely through its proprietary flow. This may be needed for A0 power-on and other early validation.
```

Comment:
`A0` needs to be defined or just removed.

Reply:
created issue